### DISCUSSION TITLE
How to open Safari instead of Chrome on an iPhone from a MAUI app?

### DISCUSSION THREAD

**Discussion by annLiu310** [dotnet/maui#21805] [Q&A]
Hello,

I'm developing a .NET MAUI+Blazor app and encountering an issue with the default browser behavior on iPhones. The app currently opens links in Chrome by default. I need to open these links in Safari instead. Could someone guide me on how to achieve this in a MAUI application? Is there a specific setting or code snippet that forces the app to open Safari External broswer for URL requests on iOS?

Any help or guidance would be greatly appreciated. Thanks in advance!

**Comment by Xyncgas** [comment]
I thought iphone has no settings for default browsers
